libs/designsystem/src/lib/components/card/directives/card-is-clickable.directive.ts

```diff
@@ -0,0 +1,16 @@
+import { Directive, OnInit } from '@angular/core';
```

Comment:
General comment: I would love if we could make this directive more generally applicable, so it could work for other elements at a later point if we need similar functionality. Therefore, I think we should try to figure out a way to separate this a bit from the card component. I have made some suggestions below in this regard.

Reply:
With these changes, a new name should be considered. Maybe 'ElementAsButtonDirective'?